src/components/checkoutoptions.jsx

```diff
@@ -0,0 +1,93 @@
+import React, {Component} from 'react';
+import { Col, Button, Form, FormGroup, Label, Input, Row, Container,} from 'reactstrap';
+import {Link} from 'react-router-dom';
+import SectionHeadingAndWhiteLine from './complibrary/sectionheadingandwhiteline';
+
+
+class CheckoutOptions extends Component {
+
+    constructor(props){
+        super(props);
+        this.state = {
+        }
+        this.checkout = this.checkout.bind(this);
```

Comment:
❕ like you did https://github.com/reetom/smartcomm/pull/1/files#diff-9348c5ff7832772118484ed0ebb58316R51
you can use the arrow method instead of bind